src/instrument.js

```diff
@@ -411,6 +411,13 @@ function unliftStore(liftedStore, liftReducer) {
  * Redux instrumentation store enhancer.
  */
 export default function instrument(monitorReducer = () => null, options = {}) {
+  if (options.maxAge !== null && options.maxAge < 2) {
```

Comment:
I’d make it `!=` to make it clear `undefined` falls into the same bucket. Even though `<` would be false anyway. If a lint rule prevents this we should surround this with disable/enable comments.